rules/catch-error-name.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const astUtils = require('eslint-ast-utils');
 const avoidCapture = require('./utils/avoid-capture');
-const getDocsUrl = require('./utils/get-docs-url');
+const getDocumentsUrl = require('./utils/get-documents-url');
```

Comment:
I think `docs` here stands for `documentation` rather than for `documents`.

Reply:
So, I should rename them again....

Reply:
I recommend using [`rg`](https://github.com/BurntSushi/ripgrep) and `perl` for batch replacements.

```bash
rg 'getDocumentsUrl' -l | xargs perl -i -pe 's/getDocumentsUrl/getDocumentationUrl/g'
rg 'get-documents-url' -l | xargs perl -i -pe 's/get-documents-url/get-documentation-url/g'
```

Reply:
done.

Reply:
@futpib I did this with vscode `replace in file`(not sure what this called, using other lang), after I saw you comment, remember me that sindresorhus has a new cli [`replace-in-files-cli`](https://github.com/sindresorhus/replace-in-files-cli)